cms/static/js/certificates/models/certificate.js

```diff
@@ -21,7 +21,10 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
             description: 'Description of the certificate',
             course_title: 'Title of the course',
             org_logo_path: '',
-            version: 1
+            version: 1,
+
+            //only one certificate is allowed to create, default set to true.
+            singleEntity: true
         },
```

Comment:
I'm fine adding a new parameter, but rather than creating one that is specific to our use case, could we instead create a parameter that holds the number of certificates allowed?  So it could be like 'maxCertificateConfigurations' and for our use case we set the value to 1